docs/user-guide.md

```diff
@@ -112,42 +120,3 @@ Then run the following to connect:
 cqlsh `docker-machine ip default` --cqlversion=3.2.0
 ```
 
-## Zipkin's internal storage structure (in Cassandra)
```

Comment:
This is great!

move this https://github.com/openzipkin/zipkin/blob/master/zipkin-cassandra-core/README.md and link to it?

cc @michaelsembwever @danchia